src/components/picker/__tests__/index.spec.tsx

```diff
@@ -37,6 +37,58 @@ describe('Picker', () => {
     jest.clearAllMocks();
   });
 
+  describe('Items', () => {
+    const testItems = [{key: 'one', value: 1, label: 'One', testID: 'item_one'}, {key: 'two', value: 2, label: 'Two', testID: 'item_two'}];
+    const testItemExists = (TestCase: () => React.JSX.Element) => {
+      const renderTree = render(<TestCase/>);
+      const driver = PickerDriver({renderTree, testID}, false);
+      driver.open();
+      const item = driver.itemDriver(testItems[0].testID);
+      expect(item.exists()).toBeTruthy();
+    };
+
+    it('should render picker items when passing children', () => {
+      const TestCase = () => {
+        return (
+          <Picker testID={testID}>
+            <Picker.Item {...testItems[0]}/>
+            <Picker.Item {...testItems[1]}/>
+          </Picker>
+        );
+      };
+      testItemExists(TestCase);
```

Comment:
In this test you are using 2 items as the Picker children.
But in the `testItemExists` function you are checking only if the first item exist.
The function should support array of the relevant items.